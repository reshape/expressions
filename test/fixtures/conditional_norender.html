<p>x</p>
<if condition='false'>
  <p>hi</p>
</if>
<p>x</p>
