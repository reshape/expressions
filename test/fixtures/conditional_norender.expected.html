<p>x</p>
<p>x</p>
